packages/nfid-sdk-react/src/molecules/accordion/index.tsx

```diff
@@ -1,26 +1,26 @@
 import clsx from 'clsx';
 import React from 'react';
 
-interface AccordionProps
+export interface AccordionProps
   extends React.DetailedHTMLProps<
     React.HTMLAttributes<HTMLDivElement>,
     HTMLDivElement
   > {
-  title: string
-  details: string | React.ReactNode
+  title: string;
+  details: string | React.ReactNode;
 }
 
 export const Accordion: React.FC<AccordionProps> = ({
   className,
   title,
   details,
 }) => {
-  const [expanded, setExpanded] = React.useState(false)
+  const [expanded, setExpanded] = React.useState(false);
 
   return (
-    <div className={clsx("", className)}>
+    <div className={clsx('', className)}>
```

Comment:
Why is it better then `className={className}`?

Reply:
It's not better. Just within the template.